components/migration-client/wso2-api-migration-client/src/main/java/org/wso2/carbon/apimgt/migration/migrator/v300/V300RegistryResourceMigrator.java

```diff
@@ -54,14 +54,16 @@ public void migrate() throws APIMigrationException {
      *
      * @throws APIMigrationException
      */
-    private void updateGenericAPIArtifacts(RegistryService registryService) {
+    private void updateGenericAPIArtifacts(RegistryService registryService) throws APIMigrationException {
+        boolean isError = false;
+        log.info("WSO2 API-M Migration Task : Started Updating API artifacts for all tenants");
         for (Tenant tenant : tenants) {
-            try {
-                registryService.startTenantFlow(tenant);
-                log.info("WSO2 API-M Migration Task : Updating APIs of tenant " + tenant.getId() +
-                        '(' + tenant.getDomain() + ')');
-                GenericArtifact[] artifacts = registryService.getGenericAPIArtifacts();
-                for (GenericArtifact artifact : artifacts) {
+            registryService.startTenantFlow(tenant);
+            log.info("WSO2 API-M Migration Task : Updating APIs of tenant " + tenant.getId() +
```

Comment:
in wso2 code style, + should comes to the next line if statement goes to more than one line.
```suggestion
            log.info("WSO2 API-M Migration Task : Updating APIs of tenant " + tenant.getId()
```